retirement_api/utils/ssa_check.py

```diff
@@ -0,0 +1,129 @@
+# utilities for checking results from SSA's Quick Calculator
+import sys
+import datetime
+from copy import copy
+import json
+
+from django.http import HttpRequest
+from .ss_calculator import get_retire_data
+from ..models import Calibration
+
+SSA_PARAMS = {
+    'dobmon': 0,
+    'dobday': 0,
+    'yob': 0,
+    'earnings': 40000,
+    'lastYearEarn': '',
+    'lastEarn': '',
+    'retiremonth': '',
+    'retireyear': '',
+    'dollars': 1,
+    'prgf': 2
```

Comment:
a good question. the param names are defined by SSA, so we can't change them. i couldn't find documentation on what that param specifies. I can see that all posts to the Quick Calculator have that value. I'll ask our contacts.